test_oss_cloud_api_compatibility.py

```diff
@@ -276,12 +279,19 @@ def test_oss_api_types_are_cloud_compatible(oss_type, cloud_schema):
                 if "workspace" in cloud_props:
                     cloud_props.remove("workspace")
 
-            assert oss_props == cloud_props
+            if isinstance(oss_props, list):
+                # when there are multiple possible path values, OSS should be a subset of Cloud
+                assert set(oss_props) <= set(cloud_props)
```

Comment:
Oh that's smart, if we do this, do we need any of the `INCLUDES_ACCOUNT_AND_WORKSPACE` stuff from before?

Reply:
It looks like the `INCLUDES_ACCOUNT_AND_WORKSPACE` logic isn't necessary with this change. I can remove it as part of this PR.